test/lib/fixtures/goof/dep-tree_small.json

```diff
@@ -1,84 +1,101 @@
 {
   "name": "goof",
   "version": "0.0.3",
```

Comment:
Can we keep the file names concistent with - instead of _? Also please can we name the trees to match the test cases? Rather than small and just dep-tree an we calle them something like:
dep-tree-no-dev-deps and dep-tree-with-dev-deps etc so it is very clear which test each one matches

Reply:
Good point, by small, I tried to suggest it's a subset of whole depTree generate by resolve-deps, but it's probably not needed anymore.